pass/add_creds.sh

```diff
@@ -107,6 +107,7 @@ help() {
   printf "gerrit\t\t\tCreate Gerrit credentials (SSH keypair).\n"
   printf "github\t\t\tCreate GitHub credentials (username/password).\n"
   printf "github_ssh\t\tCreate SSH credentials for GitHub (SSH keypair).\n"
+  printf "matrix\t\tCreate Matrix credentials for (username/password).\n"
```

Comment:
Sorry to nitpick. Text should probably be "Create Matrix credentials for chat.eclipse.org (username/password)"